platform_channels/lib/main.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/method_channel_demo.dart';
+
+void main() {
+  runApp(PlatformChannelSample());
+}
+
+class PlatformChannelSample extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      routes: {
+        '/methodChannelDemo': (context) => MethodChannelDemo(),
+      },
+      title: 'Platform Channel Sample',
+      home: HomePage(),
+    );
+  }
+}
+
+// List of all the demos with their titles and routes.
+final demoList = [
+  {
+    'demoTitle': 'MethodChannel Demo',
+    'demoRoute': '/methodChannelDemo',
+  }
+];
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Platform Channel Sample'),
+      ),
+      body: SingleChildScrollView(
```

Comment:
I did consider using a `ListView`, but I think we won't have many DemoTile's on the screen. I'm using `SingleChildScrollView` just to be on safer side in case if user is testing in small device.

Reply:
Discussed offline, and we decided to go with a ListView.